src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/PiglinListener.java

```diff
@@ -20,6 +26,18 @@
  * @author poma123
  * 
  */
+
+/**
+ * Also {@link Listener} Listens to the EntityDropItemEvent event to inject a {@link RandomMobDrop} if its dropChance() check passes.
+ * this would only be possible if the {@link RecipeType} is of PiglinBarter type. 
+ * 
+ * @author dNiym
+ * 
+ * @see getBarterDrops
+ * @see RandomMobDrop
+ * 
+ */
+
```

Comment:
You can't just add a new docs comment below the original one, edit the description instead and add yourself as an author there.